robotouille/env.py

```diff
@@ -0,0 +1,507 @@
+from typing import List, Optional, Union
+from pddlgym.backend.action import Action
+from pddlgym.backend.predicate import Predicate
+from pddlgym.backend.object import Object
+from pddlgym.backend.domain import Domain
+from pddlgym.backend.state import State
+from pddlgym.backend.special_effect import ConditionalEffect, DelayedEffect, RepetitiveEffect
+from environments.env_generator.builder import entity_to_entity_field, create_unique_and_combination_preds, create_combinations
+import copy
+
+import gym
+from gym import spaces
+import pygame
+import numpy as np
+
+STATION_FIELD = "stations"
```

Comment:
Can we move all the hardcoding to a separate file and put a TODO in there that the values will be extracted from the domain JSON once we create it? The only thing in this file should be RobotouilleEnv (you can import what you need from the other file)